enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/statemachine/StateMachineConfig.java

```diff
@@ -237,6 +232,11 @@ private void configureDocumentVerificationTransitions(StateMachineTransitionConf
                 .source(EnrollmentState.DOCUMENT_UPLOAD_VERIFICATION_PENDING)
                 .event(EnrollmentEvent.EVENT_NEXT_STATE)
                 .action(verificationDocumentStartAction)
+                .target(EnrollmentState.DOCUMENT_VERIFICATION_IN_PROGRESS)
```

Comment:
A choice state is a pseudo state, so to change `IndentityVerification` phase by interceptor, I force this transition to `_IN_PROGRESS`